MAINTAINERS

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
We should probably do a code owners file too https://github.com/blog/2392-introducing-code-owners